src/com/lftechnology/java/training/srijan/basics/football/ManchesterUnitedStats.java

```diff
@@ -0,0 +1,229 @@
+package com.lftechnology.java.training.srijan.basics.football;
+
+import java.util.logging.Logger;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+/**
+ * <p>
+ * Set up a string variable to hold the following results:
+ * </p>
+ * <p>
+ * String results =
+ * "Manchester United 1 Chelsea 0, Arsenal 1 Manchester United 1, Manchester United 3 Fulham 1, Liverpool 2 Manchester United 1, Swansea 2 Manchester United 4"
+ * ;
+ * </p>
+ * <p>
+ * Write a programme to work out how many wins Manchester United had, how many
+ * games they drew, and how many Manchester United lost.
+ * </p>
+ * <p>
+ * Extend the programme to work out how many goals Manchester United scored and
+ * how many they conceded.
+ * </p>
+ * <p>
+ * Suppose a win gains you 3 points, a draw 1 point, and a loss no points. Have
+ * your programme work out how many points in total Manchester United have
+ * acquired.
+ * </p>
+ * <p>
+ * When you have finished this exercise, the output in your textbox, listbox or
+ * console should be as follows:
+ * </p>
+ * <p>
+ * No. of wins = 3
+ * </p>
+ * <p>
+ * No. of draws = 1
+ * </p>
+ * <p>
+ * No. of defeats = 1
+ * </p>
+ * <p>
+ * Goal Scored = 10
+ * </p>
+ * <p>
+ * Goal Conceded = 6
+ * </p>
+ * <p>
+ * No. of points = 10
+ * </p>
+ * 
+ * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+ *
+ */
+public class ManchesterUnitedStats {
+	private static final Logger LOGGER = Logger.getLogger(ManchesterUnitedStats.class.getName());
+	private String[] fixtureWithScore;
+	private String clubName;
+
+	public ManchesterUnitedStats(int numberOfGameWeeks, String club) {
+		this.fixtureWithScore = new String[numberOfGameWeeks * 4];
+		this.clubName = club;
+		for (int i = 0; i < fixtureWithScore.length; i++) {
+			fixtureWithScore[i] = "";
+		}
+	}
+
+	/**
+	 * <p>
+	 * Displays Manchester United's fixtures
+	 * </p>
+	 * 
+	 * @param numberOfMatches
+	 *            {@link Integer}
+	 * @param gamesFixtures
+	 *            {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void displayGameweekFixtures(int numberOfMatches, String[] gamesFixtures) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "numberOfMatches". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)